src/js/components/Text/index.d.ts

```diff
@@ -18,7 +19,7 @@ export interface TextProps {
   size?: "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge" | string;
   tag?: PolymorphicType;
   as?: PolymorphicType;
-  textAlign?: "start" | "center" | "end";
+  textAlign?: TextAlignType;
```

Comment:
since you are already there, will you be able to place it alphabetically?

Reply:
@ShimiSun gotcha! Fixed this in this file, and I checked all other component index.d.ts files and edited those to be alphabetical also.

Reply:
girl, you ROCK!